### PR TITLE
[test-runner-junit-reporter] Add filepaths and line numbers to output

### PR DESCRIPTION
**Pull request by bennypowers**
## What I did

1. update the XML format with line numbers
2. add a test case that includes a source file (for stack traces)
3. fix tests to run properly
4. run all windows node tests against the production build

### CODE REVIEW

File: packages/test-runner-junit-reporter/test/fixtures/multiple/expected.xml [hunk 1]
```diff
@@ -1,30 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (bennypowers) [inline packages/test-runner-junit-reporter/test/fixtures/multiple/expected.xml:1]: @mikepenz Would you mind reviewing these `expected.xml` files?

**Review by Westbrook — APPROVED** [review]
I’d love to be able to see a pre release of this to ensure it still works aOK with CircleCI, but not that big of a deal to make bug fixes. 

So, LGTM!!

**Comment by bennypowers** [comment]
Maybe a patch would help?

**Comment by Westbrook** [comment]
Any word on that test failure?

**Comment by bennypowers** [comment]
ugh.. it's a windows thing

**Comment by bennypowers** [comment]
ok @Westbrook at this point the test failures are the same as on #1821 , AFAICT unrelated

**Pull request merged**